app/components/LegalModal.tsx

```diff
@@ -46,6 +46,7 @@ export const LegalModalComponent: React.SFC<InjectedFormProps & ILegalModalProps
         </div>
       </Modal.Body>
       <Modal.Footer>
+        <div className={styles.placeholder}>PLACEHOLDER</div>
```

Comment:
[Text marked with orange background] will be personalized when your funds are committed and transaction is mined. You will be able to download and print your agreement.
---------------
[this should be with this orange background]